DEVELOP.rst

```diff
@@ -2,6 +2,32 @@
 Developer Guide
 ===============
 
+
+Building from source
+====================
+
+To build the CrateDB Prometheus Adapter from source, you need to have a working
+Go environment with **Golang version 1.16** installed.
+
+Use the ``go`` tool to download and install the ``cratedb-prometheus-adapter`` executable
+into your ``GOPATH``:
+
+.. code-block:: console
+
+   go get github.com/crate/cratedb-prometheus-adapter
+   cd $GOPATH/src/github.com/crate/cratedb-prometheus-adapter
+
+Alternatively, you can clone the repository and compile the binary:
+
+.. code-block:: console
+
+   mkdir -pv ${GOPATH}/src/github.com/crate
+   cd ${GOPATH}/src/github.com/crate
+   git clone https://github.com/crate/cratedb-prometheus-adapter.git
+   cd cratedb-prometheus-adapter
+   go build
+
+
```

Comment:
```suggestion
```

This has been taken verbatim from "README.rst". Given that within "DEVELOP.rst", there is already a section about this, I believe it will be safe to remove this?